cmd/ocm/gcp/update-wif-config.go

```diff
@@ -103,6 +110,29 @@ func updateWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		wifBuilder.WifTemplates(append(existingTemplates, wifTemplate)...)
 	}
 
```

Comment:
It is not necessary to pass the federated project on every update call.

You should set it only if the user provided the federated-project option and provided a value that is different from the existing value in the wif-config